.github/workflows/configure.yml

```diff
@@ -12,11 +12,6 @@ on:
 permissions:
   contents: read
 
-env:
-  PIP_BREAK_SYSTEM_PACKAGES: 1
-  # For cmake:
-  VERBOSE: 1
```

Comment:
Could we please keep the `VERBOSE`?

If things go well, usually we don't look at the logs (and don't care how verbose they are).

If things go wrong it's often a pain if the full context isn't available. It's easy to ignore extra information. It's often impossible to guess what's missing.

Reply:
This does nothing in this job. The job only configures, while this would cause the compile lines to be emitted in the build step. There's no build step.